assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -142,7 +142,8 @@ che.oauth.github.clientsecret=NULL
 che.oauth.github.authuri= https://github.com/login/oauth/authorize
 che.oauth.github.tokenuri= https://github.com/login/oauth/access_token
 che.oauth.github.redirecturis= http://localhost:${SERVER_PORT}/wsmaster/api/oauth/callback
-
+# register github even without client id and secret
+che.oauth.github.forceactivation=false
```

Comment:
These video shows an attempt to perform a git operation or github operation (would be similar to clicking in OAuth button):
1. attempt with default che.oauth.github.forceactivation false
2. attempt with che.oauth.github.forceactivation true without setToken
3. attempt with che.oauth.github.forceactivation true and setting the token

- Github operation (upload ssh key - old demo, but will show what happen) https://youtu.be/y9cux67RQHY It shows a 404 in a pop in one case + notification error of Oauth failure
- Git operation (push) https://youtu.be/5D6XZlDyPKE . It shows a notification error